spotinst_sdk2/clients/ocean_cd/__init__.py

```diff
@@ -0,0 +1,950 @@
+        provider_update (RolloutSpec): Rollout Spec Object
+
+        # Returns
+        (Boolean): Response Status
+        """
+        request = rollout_spec.RolloutSpecRequest(provider_update)
+
+        excluded_node_update_dict = self.exclude_missing(
+            json.loads(request.toJSON()))
+
+        formatted_node_update_dict = self.convert_json(
+            excluded_node_update_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_node_update_dict)
+
+        response = self.send_put(
+            body=body_json,
+            url=self.__base_rollout_spec_url + "/" + name,
+            entity_name=self.ENTITY_NAME5)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["status"]
+
+    def delete_oceancd_rollout(self, name: str):
+        """
+        Delete an existing Ocean CD Rollout Spec.
+
+        # Arguments
+        name (String): OceanCD Rollout Spec
+
+        # Returns
+        (Boolean): Response Status
+        """
+
+        response = self.send_delete(
+            url=self.__base_rollout_spec_url + "/" + name,
+            entity_name=self.ENTITY_NAME5)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["status"]
+
+    def describe_rollout_by_id(self, rollout_id: str):
+        """
+        Get Ocean CD single rollout
+
+        # Arguments
+        rollout_id (String): The identifier of the Ocean CD rollout
+
+        # Returns
+        (Object): OceanCD Rollout List API response
+        """
+        response = self.send_get(
+            url=self.__base_rollout_url + "/" + rollout_id,
+            entity_name=self.ENTITY_NAME6)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+
+    def ocean_cd_rollout_actions(self, rollout_id: str, action_update: rollout.Action):
+        """
+        Execute action on an existing Ocean CD rollout.
+
+        # Arguments
+        rollout_id (String): The identifier of the Ocean CD rollout
+
+        # Returns
+        (Boolean): Response Status
+        """
+        request = rollout.ActionRequest(action_update)
+
+        excluded_node_update_dict = self.exclude_missing(
+            json.loads(request.toJSON()))
+
+        formatted_node_update_dict = self.convert_json(
+            excluded_node_update_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_node_update_dict)
+
+        response = self.send_put(
+            body=body_json,
+            url=self.__base_rollout_url + "/" + rollout_id,
+            entity_name=self.ENTITY_NAME6)
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
```

Comment:
PLease confirm that useful information is in the items, probably this function should return status